segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/Storage.java

```diff
@@ -196,6 +196,22 @@
      */
     CompletableFuture<Iterator<SegmentProperties>> listSegments();
 
+    /**
+     * Indicates a beginning of the iteration or batch of operations.
+     *
+     * @param iterationId identifier of iteration.
+     */
+    default void beginIteration(long iterationId) {
```

Comment:
This method is not at all obvious in what it does based on its documentation here.

Reply:
Fixed